cmake/mcuboot.cmake

```diff
@@ -0,0 +1,246 @@
+# Included by mcuboot/zephyr/CMakeLists.txt
+set(MCUBOOT_BASE ${CMAKE_CURRENT_LIST_DIR}/../../mcuboot/)
```

Comment:
I suggest we make the following change to Zephyr and upstream the change seen here:
https://github.com/NordicPlayground/fw-nrfconnect-zephyr/pull/248

That way, this code can be changed into:
```suggestion
set(MCUBOOT_BASE ${ZEPHYR_MCUBOOT_MODULE_DIR})
```
or simply use `ZEPHYR_MCUBOOT_MODULE_DIR` in code below and avoid the need for `MCUBOOT_BASE`

Reply:
Done :)